### DISCUSSION TITLE
How to get the key prop of selected dropdown option?

### DISCUSSION THREAD

**Discussion by asimhafeezz** [react-hook-form/react-hook-form#1733] [General] [answered]
getting Select list dynamically and trying to get the specifically selected key prop of options in onChange Handler.
on console.log(selected.target) i only get the selected value and name... is it possible to get key prop?
![rhf](https://user-images.githubusercontent.com/56485049/83315438-6fc15c00-a239-11ea-8948-073d6a7b8e6a.png)
i want to access key prop of selected <MenuItem />
![val](https://user-images.githubusercontent.com/56485049/83315448-7cde4b00-a239-11ea-9a28-2e44de7e7096.png)

this is (selected.target) value

**Comment by bluebill1049** [comment, marked as answer]
hmmm i don't think you get the key from the Select component.

**Reply by asimhafeezz** [reply]
ohh... then is there anyway to get the {item.productcategory_ID} which i am passing to key prop? of selected one :/

**Reply by bluebill1049** [reply]
can't you do a index lookup?

**Reply by asimhafeezz** [reply]
yess..thanku it's done <3.
